### ISSUE TITLE
Report error instead of silently failing

### ISSUE THREAD

**Issue by rmccue** [humanmade/S3-Uploads#179]
If S3 Uploads decides not to load, we should trigger an error (`E_USER_WARNING`) rather than silently bailing out.

**Comment by AaronHolbrook** [comment]
Agreed; for example, activating the plugin without credentials should not cause a fatal error
